Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,33 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "";
+            //string sureName = "";
+            int age = 0;
+            float weight = 0f;
+            float height = 0f;
+
+            Console.Write("Name: ");
+            name = Console.ReadLine();
+
+            //Console.Write("Surename: ");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4864232)

Reply:
okay, done!